src/screens/Key/Enter/index.js

```diff
@@ -179,7 +199,17 @@ class EnterKeyScreen extends NavigatorComponent<Actions & KeyState & Props, Stat
   }
 
   onDonePressed = () => {
-    this.props.validateMnemonic();
+    if (this.props.isOnResetPassProcess) {
```

Comment:
Please use `this.props.isOnResetPassProcess === true` to prevent coercion.

Reply:
done